examples/h5dsm_attr_create.c

```diff
@@ -18,14 +18,11 @@ int main(int argc, char *argv[]) {
     if(0 != uuid_parse(argv[1], pool_uuid))
         ERROR;
 
-    /* Initialize VOL */
-    if(H5daos_init(pool_uuid, pool_grp, getenv("DAOS_SVCL") ? getenv("DAOS_SVCL") : "0") < 0)
-        ERROR;
-
     /* Set up FAPL */
     if((fapl = H5Pcreate(H5P_FILE_ACCESS)) < 0)
         ERROR;
-    if(H5Pset_fapl_daos(fapl, MPI_COMM_WORLD, MPI_INFO_NULL) < 0)
+    if(H5Pset_fapl_daos(fapl, pool_uuid, pool_grp, getenv("DAOS_SVCL") ? getenv("DAOS_SVCL") : "0",
```

Comment:
the svcl is not there anymore; actually removed from the DAOS API.
it's probably better to just remove it from this function.
For backward compatibility with DAOS 1.0, we can just the svcl to DAOS_SVCL env just where we set the pool connect args here:
https://github.com/HDFGroup/vol-daos/blob/master/src/daos_vol.c#L1601

Reply:
Sounds reasonable to me. I'm definitely in favor of removing it from the API then so that we don't have to deprecate it later.